### DISCUSSION TITLE
Types of item in the Second Renaissance wiki

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#949] [Research]
As we are researching organisations, we are coming across multiple types of items that we might like to capture in separate, linked notes (instead of just in free text of organisation profile):

- Leading figures (individuals) so that we can see who are the influential figures/key thinkers in the field (and what clusters and patterns)
- Resources
  - so that we can generate a table view of resources which would be useful for later work packages WP4 and WP5 (a table view of resources categorised by a few key fields which I tend to note in free text form anyway (e.g. type, author, summary) seems a lot more useful than someone having to comb through org profiles to pull out and organise the resources later 
  - **Possibly nice to have rather than essential, but I think it would be both valuable and make the data collection phase more easy to have structured fields to fill in where appropriate/easy for resources**
- Frameworks/methodologies/concepts
  - so that we can see what are the key influential framewroks/methodologies/concepts in the field (and what clusters/patterns)

So far we only have a template for capturing data on organisations.

## Questions 

- (How) do we want to distinguish between the 4 different types of item? (organisation, resource, leading figure, framework) **what's the so that ...?**
  - Hyp: add a field to existing template to flag what type of item it is? **Yes, this would be entitytype: org | person | ...**
  - Do we put all notes in the same folder, e.g. profiles, or separate by type e.g. a folder for org profiles, folder for resources etc.?
- Do we want to create some kind of templates for the other 3 types of item? **Doesn't seem necessary for leading figures and frameworks, see brainstorm below. Think can be helpful for resources so that I can just put a title in the org profile note and then link to a separate note for the resource where I fill in some standard information fields (added bonus that we can generate a table view of resources at the end)**
  - Hyp: create a template for resources with new frontmatter fields?

## Brainstorm of what (additional) fields to include

Leading figures/individuals 
- link e.g. to wikipedia
- paste some description
- backlinks

Resources
- type, e.g. book, video, article, reading list
- author, creator or editor
- link
- summary
- topic
- notes
- backlinks
- date published

Frameworks/methodologies/concepts
- link
- paste some description
- backlinks

**Comment by matthewmccarthy11** [comment]
In response to the first part here: For the time being, Catherine will use her best judgement to guide which of the items will be put into the 'stub' , and we want to keep these stubs simple. Just a link and very small description is fine.

**Comment by rufuspollock** [comment]
@catherinet1 I would really consider the YAGNI principle in the first pass here. Do we really need to distinguish these, if so, why and why *now*?

Note there are at least a few different questions here that may get mixed up e.g.

- Do we want to include e.g. Leading Figures. (Sure, why not since just another wiki page for now)
- Do we need some way to *distinguish* Leading Figures at the metadata level? Hmmm, not sure we really need to and leading to a bunch of ontological naval gazing (e.g. distinguishing figures from frameworks etc etc)
- What special metadata do we need etc ... (don't get distracted by this for now IMO)

Overall, we want to keep laser focused on a use case (or two) e.g. on creating the map that helps us sensemake about the space ...

**Comment by asimong** [comment]
Maybe worth revisiting this now … I'm happy to!
